lib/content-services/src/lib/category/categories-management/categories-management.component.spec.ts

```diff
@@ -418,13 +391,11 @@ describe('CategoriesManagementComponent', () => {
             expect(categoriesChangeSpy).toHaveBeenCalledOnceWith(component.categories);
         }));
 
-        it('should clear and hide input after category is created', fakeAsync(() => {
-            const controlVisibilityChangeSpy = spyOn(component.categoryNameControlVisibleChange, 'emit');
+        it('should clear and not hide input after category is created', fakeAsync(() => {
```

Comment:
no need checking the visibility for this test, you can just call it "should clear input after category is created" and remove check for visibilty, unless you expect it to be hidden